src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageAnalyzerResultProvider.cs

```diff
@@ -16,16 +16,19 @@ namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
 {
     public class PackageAnalyzerResultProvider : IAnalyzeResultProvider
     {
-        private const string Id = "UA101";
+        private const string RuleId = "UA101";
+        private const string RuleName = "SomeRuleName";
+        private const string FullDescription = "Some full description";
+        private readonly Uri _helpUri = new("about:blank");
```

Comment:
For now the RuleName and Full Description can be the same as ToolName and the helpUri can be the same as InformationUri.

Reply:
just updated.